schedule/yast/sle_15_sp3/autoyast_create_hdd_maintenance_deregister.yaml

```diff
@@ -1,23 +1,51 @@
 ---
-name: autoyast_create_hdd_maintenance
+name: autoyast_create_hdd_maintenance_deregister
```

Comment:
Ok, create a new file in yast/maintenance/  and named as 'schedule/yast/maintenance/autoyast_create_hdd_maintenance_deregister_s390x.yaml'.

Reply:
I'm not sure if this schedule is compatible with other product, if not, we need to put a suffix with the product, but for sure we need to name if first with `ha_` and remove deregister, because it is the only way to do it right:
`ha_autoyast_create_hdd_maintenance_<arch/backend>.yaml (<arch/backend in they way that other files are created in the repo)